schema.sql

```diff
@@ -6,5 +6,24 @@ CREATE TABLE animals (
     date_of_birth date not null,
     escape_attempts int not null,
     neutered boolean not null,
-    weight_kg decimal not null
+    weight_kg decimal not null,
+    species varchar(100)
 );
+
+-- Create the owners table
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    full_name VARCHAR(255),
+    age INTEGER
+);
+
+-- Create the species table
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255)
+);
+
+-- Modify the animals table
+ALTER TABLE animals
+ADD COLUMN species_id INTEGER REFERENCES species(id),
```

Comment:
- [ ] We want to expand the _species_ column found in _animals_ table by creating a _species_ table. Therefore we need to remove the _species_ column of from _animals_ table and rather link _species_ table to _animal_ table using a foreign key. 

     Good job for creating the _species_ table and adding foreign key reference, therefore let's remove the _species_ column from the _animals_ table by adding this query just after _line 27_:

```sql

REMOVE species;

```